src/main/java/lotto/model/MatchingResult.java

```diff
@@ -0,0 +1,27 @@
+package lotto.model;
+
+public enum MatchingResult {
+    THREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 1500000),
+    SIX(6, 2000000000);
```

Comment:
```suggestion
    SIX(6, 2_000_000_000);
```
언더스코어를 활용하여 가독성을 높힐 수 있습니다!